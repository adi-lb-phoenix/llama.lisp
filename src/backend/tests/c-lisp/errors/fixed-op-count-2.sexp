(c-lisp
    (define ((main int))
        (ret (add 3 4 5))))
