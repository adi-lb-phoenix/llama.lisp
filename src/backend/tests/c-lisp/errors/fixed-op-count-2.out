Error in statement: ['ret', ['add', 3, 4, 5]]
CodegenError: `add` takes only 2 operands: ['add', 3, 4, 5]
